bluebrain/repo-bluebrain/packages/py-atlas-building-tools/package.py

```diff
@@ -11,48 +11,23 @@ class PyAtlasBuildingTools(PythonPackage):
     homepage = "https://bbpgitlab.epfl.ch/nse/atlas-building-tools"
     git      = "git@bbpgitlab.epfl.ch:nse/atlas-building-tools.git"
 
-    version('0.1.8', tag='atlas-building-tools-v0.1.8')
-    version('0.1.7', tag='atlas-building-tools-v0.1.7')
-    version('0.1.6', tag='atlas-building-tools-v0.1.6')
-    version('0.1.5', tag='atlas-building-tools-v0.1.5')
-    version('0.1.4', tag='atlas-building-tools-v0.1.4')
-    version('0.1.3', tag='atlas-building-tools-v0.1.3')
-    version('0.1.2', tag='atlas-building-tools-v0.1.2')
-    version('0.1.1', tag='atlas_building_tools-v0.1.1')
+    version('0.1.9', tag='atlas-building-tools-v0.1.9')
 
-    depends_on('py-setuptools', type=('build', 'run'))
-    depends_on('py-cached-property', type=('build', 'run'))
-    depends_on('py-cgal-pybind@0.1.1:', type=('build', 'run'), when='@0.1.2:')
-    depends_on('py-cgal-pybind@0.1.0:', type=('build', 'run'), when='@0.1.1')
-    depends_on('py-click@7.0:', type=('build', 'run'))
-    depends_on('py-networkx@2.4:', type=('build', 'run'))
-    depends_on('py-nptyping@1.0.1:', type=('build', 'run'))
-    depends_on('py-numba', type=('build', 'run'))
-    depends_on('py-numpy@1.15.0:', type=('build', 'run'))
-    # numpy-quaternion version is capped because of an issue similar to
-    # https://stackoverflow.com/questions/20518632/importerror-numpy-core-multiarray-failed-to-import
-    depends_on('py-numpy-quaternion', type=('build', 'run'))
-    depends_on('py-openpyxl@3.0.3:', type=('build', 'run'))
-    depends_on('py-pandas@1.0.3:', type=('build', 'run'))
+    depends_on('py-atlas-densities@0.1.1:', type=('build', 'run'))
+    depends_on('py-atlas-direction-vectors@0.1.1:', type=('build', 'run'))
+    depends_on('py-atlas-placement-hints@0.1.1:', type=('build', 'run'))
+    depends_on('py-atlas-splitter@0.1.1:', type=('build', 'run'))
+    depends_on('py-cgal-pybind@0.1.1:', type=('build', 'run'))
```

Comment:
According to setup.py:
```suggestion
    depends_on('py-cgal-pybind@0.1.4:', type=('build', 'run'))
```